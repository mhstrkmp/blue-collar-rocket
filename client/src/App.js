import React from "react";
import Page from "./pages/Page";
import "./App.css";

function App() {
  return (
    <div className="App">
      <Page
        onLogin={() => console.log("login")}
        onLogout={() => console.log("logout")}
        onCreateAccount={() => console.log("create account")}
      />
      <a href="/storybook" target="_blank">
        Go to Storybook
      </a>
    </div>
  );
}

export default App;
