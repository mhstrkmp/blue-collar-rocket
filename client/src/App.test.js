import React from "react";
import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders storybook react link", () => {
  render(<App />);
  const linkElement = screen.getByText(/Go to Storybook/i);
  expect(linkElement).toBeInTheDocument();
});
