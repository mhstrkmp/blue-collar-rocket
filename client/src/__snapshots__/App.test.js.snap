// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders 1`] = `
Array [
  <header
    className="Header__Container-sc-17wre1r-0 bkfIWN"
  >
    <h1
      className="Header__Title-sc-17wre1r-1 dMDOcO"
    >
      Home
    </h1>
  </header>,
  <section
    className="Wrapper__ContentWrapper-sc-1so8v63-0 ivINYv"
  />,
  <a
    href="/storybook"
    target="_blank"
  >
    Go to Storybook
  </a>,
  <div
    className="NavbarBottom__Container-okzrgt-0 dRgMHa"
  >
    <a
      aria-current="page"
      className="NavbarBottom__StyledNavLink-okzrgt-1 gBInXk active"
      href="/"
      onClick={[Function]}
      style={
        Object {
          "fill": "var(--primary-third)",
        }
      }
    >
      <svg>
        home.svg
      </svg>
    </a>
    <a
      aria-current={null}
      className="NavbarBottom__StyledNavLink-okzrgt-1 gBInXk"
      href="/product-overview"
      onClick={[Function]}
    >
      <svg>
        find_in_page.svg
      </svg>
    </a>
    <a
      aria-current={null}
      className="NavbarBottom__StyledNavLink-okzrgt-1 gBInXk"
      href="/checkout"
      onClick={[Function]}
    >
      <svg>
        shopping_cart.svg
      </svg>
    </a>
    <a
      aria-current={null}
      className="NavbarBottom__StyledNavLink-okzrgt-1 gBInXk"
      href="/profile"
      onClick={[Function]}
    >
      <svg>
        profile.svg
      </svg>
    </a>
  </div>,
]
`;
